tests/actions_tkctl.go

```diff
@@ -0,0 +1,121 @@
+package tests
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"strings"
+	"text/template"
+
+	"github.com/pingcap/tidb-operator/tests/slack"
+	"k8s.io/kubernetes/test/e2e/framework/job"
+)
+
+var waitCheckArgsTemplate = `
+use {{ .clustername }}
+list --namespace={{- .namespace }}
+info -t {{ .clustername }}
+get all
+`
+
+func Int32Ptr(i int32) *int32 { return &i }
```

Comment:
You can use k8s utils package https://github.com/kubernetes/utils/blob/master/pointer/pointer.go#L50 directly